Automattic/sensei
# Add WPML settings tab

## Proposed Changes

* Add WPML settings tab. Show it only when WPML is active.
* Add `Use WPML slug translation` option.
* Register CPTs with slugs without translation when the option is enabled (and WPML is active) to let WPML translate slugs.

UPDATE - UI and setting name was changed as per https://github.com/Automattic/sensei/pull/7569#discussion_r1548033057.

## Testing Instructions

1. Install and activate WPML. 
2. Create a course in English.
3. Translate it to Spanish. 
4. Make sure you get 404 for the Spanish course on the frontend.
5. Go to Sensei LMS -> Settings -> WPML.
6. Enable WML slug translations. Save.
7. Go to the frontend and make sure the Spanish course works now.

## Screenshot
![Screenshot 2024-04-10 at 8 58 36 AM](https://github.com/Automattic/sensei/assets/1190420/a6a3c4fb-f068-4b0b-8ca0-18a255edf2dd)

## Pre-Merge Checklist
